src/renderer/components/poolActionsHistory/PoolActionsHistory.stories.tsx

```diff
@@ -141,11 +142,21 @@ export const History: Story<{ dataStatus: RDStatus }> = ({ dataStatus }) => {
   const res = useMemo(() => getResults(dataStatus), [dataStatus])
   const [currentPage, setCurrentPage] = useState(1)
   const [filter, setFilter] = useState<Filter>('ALL')
-  return (
-    <PoolActionsHistory
+  const HeaderContent = (
+    <WalletPoolActionsHistoryHeader
       availableFilters={['ALL', 'SWITCH', 'DEPOSIT', 'SWAP', 'WITHDRAW', 'DONATE', 'REFUND']}
       currentFilter={filter}
       setFilter={setFilter}
+      openViewblockUrl={() => {
+        console.log('open viewblock')
```

Comment:
console.log

Reply:
Not needed for a story - just to know `openViewblockUrl` has been called.